build.gradle

```diff
@@ -348,79 +342,101 @@ subprojects {
-                }
-            }
-            repository(url: stagingUrl, configureAuth)
-            snapshotRepository(url: 'https://oss.sonatype.org/content/repositories/snapshots/', configureAuth)
-        }
-    }
-    uploadArchives.onlyIf { !name.contains("grpc-gae-interop-testing") && !name.contains("grpc-xds") }
+    publishing {
+        publications {
+            mavenJava(MavenPublication) {
+                from components.java
 
-    [
-        install.repositories.mavenInstaller,
-        uploadArchives.repositories.mavenDeployer,
-    ]*.pom*.whenConfigured { pom ->
-        pom.project {
-            name "$project.group:$project.name"
-            description project.description
-            url 'https://github.com/grpc/grpc-java'
-
-            scm {
-                connection 'scm:git:https://github.com/grpc/grpc-java.git'
-                developerConnection 'scm:git:git@github.com:grpc/grpc-java.git'
-                url 'https://github.com/grpc/grpc-java'
-            }
+                artifact javadocJar
+                artifact sourcesJar
 
-            licenses {
-                license {
-                    name 'Apache 2.0'
-                    url 'https://opensource.org/licenses/Apache-2.0'
-                }
-            }
+                pom {
+                    name = "$project.group:$project.name".toString()
+                    url = 'https://github.com/grpc/grpc-java'
+                    afterEvaluate {
+                        // description is not available until evaluated.
+                        description = project.description
+                    }
+
+                    scm {
+                        connection = 'scm:git:https://github.com/grpc/grpc-java.git'
+                        developerConnection = 'scm:git:git@github.com:grpc/grpc-java.git'
+                        url = 'https://github.com/grpc/grpc-java'
+                    }
 
-            developers {
-                developer {
-                    id "grpc.io"
-                    name "gRPC Contributors"
-                    email "grpc-io@googlegroups.com"
-                    url "https://grpc.io/"
-                    // https://issues.gradle.org/browse/GRADLE-2719
-                    organization = "gRPC Authors"
-                    organizationUrl "https://www.google.com"
+                    licenses {
+                        license {
+                            name = 'Apache 2.0'
+                            url = 'https://opensource.org/licenses/Apache-2.0'
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id = "grpc.io"
+                            name = "gRPC Contributors"
+                            email = "grpc-io@googlegroups.com"
+                            url = "https://grpc.io/"
+                            // https://issues.gradle.org/browse/GRADLE-2719
```

Comment:
This may now be out-of-date.

Reply:
yes. the issue is marked as closed. removing the note.